applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -0,0 +1,343 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.applicationconfig;
+
+import com.azure.applicationconfig.credentials.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.azure.common.exception.ServiceRequestException;
+import com.azure.common.http.rest.Response;
+
+import java.util.List;
+
+
+/**
+ * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
+ * Store. Operations allowed by the client are adding, retrieving, updating, and deleting ConfigurationSettings, and
+ * listing settings or revision of a setting based on a filter.
+ *
+ * <p><strong>Construct the client, for additional ways to build a client see {@link ConfigurationClientBuilder here}.</strong></p>
```

Comment:
I commented either here (or for the async case) that this line should be simpler.